main.js

```diff
@@ -36,30 +36,43 @@ module.exports = (glob, options) => {
   let eXecutable = options.electron;
   if(eXecutable && fs.existsSync(eXecutable)) {
     let appPath = app.getAppPath();
-    // eslint-disable-line
+
+		// console.log('electron-reload: appPath:', appPath);
+		// console.log('electron-reload: __dirname:', __dirname);
+
+		// eslint-disable-line
     let config = require(path.join(appPath, 'package.json'));
-    let mainFile = path.join(appPath, config.main);
+    let mainFile = path.join(mainDirname, config.main);
 
-    chokidar.watch(mainFile).on('change', () => {
-      // Detaching child is useful when in Windows to let child
-      // live after the parent is killed
-      let child = spawn(eXecutable, [appPath], {
-        detached: true,
-        stdio: 'inherit'
-      });
-      child.unref();
-      // Kamikaze!
+		// console.log('electron-reload watching mainfile:',mainFile);
 
-      // In cases where an app overrides the default closing or quiting actions
-      // firing an `app.quit()` may not actually quit the app. In these cases
-      // you can use `app.exit()` to gracefully close the app.
-      if(opts.hardResetMethod === 'exit'){
-          app.exit();
-      } else {
-          app.quit();
-      }
-    });
-  } else {
+		if (fs.existsSync(mainFile)) {
```

Comment:
From this block on, the whole formatting has changed and is not possible to see the actual changes.